doc/how_to/notebook/other_nb.md

```diff
@@ -8,6 +8,12 @@ This guide addresses how to develop apps in Google Colab, VSCode, nteract, and o
 
 In the Google Colaboratory notebook, first make sure to load the `pn.extension()`. Panel objects will then render themselves if they are the last item in a notebook cell. Please note that in Colab rendering for each notebook cell is isolated, which means that every cell must reload the Panel extension code separately. This will result in somewhat slower and larger notebook than with other notebook technologies.
 
+* Be aware: using Panel with ipywidgets in Colab does not work, due to Colab's architecture constraints.
+
+### Anaconda Cloud
+
+When using Anaconda Cloud, make sure to upgrade the Panel package to the latest version using !pip install
```

Comment:
I think we should have a separate explanation on upgrading Panel to link to.

It should 1) Describe how to upgrade with pip and conda 2) recommend upgrading `param` and `bokeh` together with Panel 3) recommend optionally upgrading `hvplot` and `holoviews` together with Panel. 4)  Describe how to hard refresh the browser if no output is displayed 5) remember users to upgrade the jupyter lab python environment at the same time.

Reply:
Agree. People that start out need to be aware of the above at earliest opportunity, otherwise they will run into silent fails / confusion.

That's why I feel strongly that, as wel as documenting it, Panel should check itself and its environment after initializing, and emit the appropriate warning in the output. That seems the surest way to alert users.

So I guess a warning that is generated when doing import panel . Should that be a separate PR?

Reply:
Are you suggesting we check (and warn if we are not on the latest version) for the latest version of Panel while importing it?

Reply:
Yes absolutely! Just makes sense. And (separately(?) a warning regarding ipywidgets as indicated above.

Reply:
The real problem here continues to be lack of versioning for our docs.

Reply:
@jbednar Totally agree that it shouldn't break older apps. I am just talking about generating a warning in output / console. Could that in itself causes issues?

With "checking on import" I meant:

-import panel as pn -> Produces warning in output that "... Your Panel version is x, while latest version is y".

Would such a warning in any way cause issues, or impede an app from still working on an older version?

In that case, only show the warning on import panel as pn is the last statement in a file or Notebook cell?

Or only in console or debugger?

Totally agree apps should stay working on older version.

I did NOT mean: do import panel as pn, then anything else happens, besides showing a warning.

Do you agree there is an added value to such a warning?

Advantages:

-Informs users there is a new version available.
-Incentivizes them to look at new version and features of Panel when they revisit their old app after a while.
-Could encourage users to upgrade, and therefore reduce the installed base of older Panel versions that are more and more obsolete.

Separately, I have proposed some changes earlier to the docs to provide a heads-up on this for (new) users. 

Maybe add something in addition to the getting started / installation like: "New Panel versions are released regularly. Please be aware of this, and consider testing and upgrading your app to take advantage of new features and improvements".

One question is: do most users of Panel already know there are frequent versions changes, and that they should take that into consideration?

-People with a Python background: probably.
-Data Science background: maybe not so much?
-Users new to Panel: ?
-Other (future) target groups?

I don't suppose there is any data on the main groups of people using Panel?

Another approach: would it be easy to do Github queries that check, for example, how many old forks of Panel exist? Don't know if that would be a useful metric, but maybe another Github query might reveal insights? Or volume of !pip install vs. !pip install -U over the years? Sorry if this is silly / wrong / irrelevant :-)

Reply:
My main point is that users should be alerted in any way possible to things like the Colab issue / older versions, rather then have them fail silently, or with errors that do not make clear what the root cause is.